Mathlib/FieldTheory/IsPerfectClosure.lean

```diff
@@ -0,0 +1,509 @@
+/-
+Copyright (c) 2024 Jz Pan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jz Pan
+-/
+import Mathlib.FieldTheory.PurelyInseparable
+import Mathlib.FieldTheory.PerfectClosure
+
+/-!
+
+# `IsPerfectClosure` predicate
+
+This file contains `IsPerfectClosure` which asserts that `L` is a perfect closure of `K` under a
+ring homomorphism `i : K →+* L`, as well as its basic properties.
+
+## Main definitions
+
+- `pNilradical`: the `p`-nilradical of a ring is an ideal consists of elements `x` such that
```

Comment:
```suggestion
- `pNilradical`: the `p`-nilradical of a ring is an ideal consisting of elements `x` such that
```
Also the phrasing here and the docstring of `pNilradical` should be clarified to better into account the cases `1 < p` or not.